gst/gst_element.go

```diff
@@ -501,3 +501,21 @@ func (e *Element) URIHandler() URIHandler {
 func (e *Element) RemovePad(pad *Pad) bool {
 	return gobool(C.gst_element_remove_pad(e.Instance(), pad.Instance()))
 }
+
+// GetRequestPad gets a request pad from the element based on the name of the pad template.
+// Unlike static pad, request pads are not automatically but are only created on demand.
```

Comment:
"are not automatically **created,** but instead, only on demand"

Or something like that, sounds better

Reply:
Thanks for your comment. I supplement description.
How about following?

// GetRequestPad gets a request pad from the element based on the name of the pad template.
// Unlike static **pads**, request pads **are not created automatically but are only created on demand
// For example, audiomixer has sink template, 'sink_%u', which is used for creating multiple sink pads on demand so that it performs mixing of audio streams by linking multiple upstream elements on it's sink pads created on demand.**